docs/CLOUD.md

```diff
@@ -0,0 +1,33 @@
+# Cloud
+
+We are using the Google Compute Jenkins plugins.
+
+See https://github.com/jenkinsci/google-compute-engine-plugin/#configuration-as-code-support for further examples.
+
+## How to
```

Comment:
IMHO we need to add a section here explaining what exactly this is and why somebody would want to configure it. The purpose of setting up GCE locally is not clear from reading this documentation.

Reply:
I added the why in the description of this PR, how do you foresee we can improve this?